plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1059,6 +1059,10 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                         protocolEntry);
 
                 transactionLogWriter.flush();
+
+                if (replaceExistingTable) {
```

Comment:
do we need here as well the check to see whether the table existed before?

Reply:
I think it may be redundant considering that on [line 968](https://github.com/trinodb/trino/blob/e70495094d929e7465a093773f5f6a72a82fd37f/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java#L968)
`boolean replaceExistingTable = tableHandle != null && saveMode == SaveMode.REPLACE;`
and that we know it exists in that scenario, as we then do this on [line 970](https://github.com/trinodb/trino/blob/e70495094d929e7465a093773f5f6a72a82fd37f/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java#L970)
`ConnectorTableMetadata existingTableMetadata = getTableMetadata(session, tableHandle);`